male-seed-beetle/scripts/convert-with-core.py

```diff
@@ -1,7 +1,27 @@
 import polars as pl
+import re
+from pathlib import Path
 
-df = pl.read_csv('../data-raw/data.csv')
+#Transform from tab sep to comma sep
+with open('../data-raw/data.tsv', 'r') as tsv:
 
-#Test that it works - remove later!
-first_row = df.head(1)
-print(first_row)
+    with open('../data-raw/data1.csv', 'w') as csv:
+
+        for line in tsv:
+
+            content = re.sub("\t", ",", line) 
+
+            csv.write(content)
```

Comment:
Remove these empty lines above, they aren't necessary and make the code a bit harder to read :relaxed: